index.js

```diff
@@ -252,7 +242,7 @@ class AmazonAdsFormatter {
     let formatted = postal
       .toLowerCase()
       .replace(/[^a-z0-9]/g, "")
-      .trim();
```

Comment:
Already this is handled below no?

Reply:
removing it from below

Reply:
substring(0, -1) is treated as substring(0, 0), which results in an empty string.
@anantjain45823 have unit tests

Reply:
We now have the unit test cases now accordingly